tests/test_scoring_enterprise_issue_integer_callback_boundary.py

```
@@ -0,0 +1,219 @@
+from fast_mlsirm.scoring import AssessmentSpecError
+from fast_mlsirm.scoring.enterprise_issue import (
+    EnterpriseAssertionKind,
+    EnterpriseSourceRecord,
+    EvidenceSpanRecord,
+)
+
+
+class _HostileIndex:
+    """Integer protocol provider that records any caller callback execution."""
+
+    callbacks: list[str] = []
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)